src/url-loader/url-resolver.ts

```diff
@@ -65,3 +70,25 @@ export abstract class UrlResolver {
     return url as ResolvedUrl;
   }
 }
+
+async function inferComponentDirname(dirname: string): Promise<string> {
+  try {
+    const contents = await fs.readFile(
+        pathlib.join(dirname, '.bowerrc'), {encoding: 'utf8'});
+    const bowerrc = JSON.parse(contents);
```

Comment:
I think said library (bower-something), parses it against default options too...